src/main/kotlin/racingcar/domain/RacingCar.kt

```diff
@@ -1,18 +1,30 @@
 package racingcar.domain
 
-private const val INIT_COUNT = 0
+private const val START_POSITION = 0
 private const val MOVE_FORWARD_START_NUM = 4
 private const val MOVE_FORWARD_COUNT = 1
+private const val NAME_LENGTH_LIMIT = 5
 
 data class RacingCar(
     val name: String,
-    val count: Int = INIT_COUNT
+    val position: Int = START_POSITION
 ) {
 
+    init {
+        if (name.length > NAME_LENGTH_LIMIT) {
+            println("$name : 자동차 이름은 ${NAME_LENGTH_LIMIT}글자를 초과하여 입력할 수 없습니다.")
+            throw IllegalArgumentException()
```

Comment:
콘솔로 출력하는 이 메시지를 예외 메시지로 주입하면 될 것 같아요. 😃

RacingCar는 Model의 영역이지 View의 영역이 아니기 때문에 출력을 모델에서 수행하면 안될 것 같습니다. 👀

```suggestion
            throw IllegalArgumentException("$name : 자동차 이름은 ${NAME_LENGTH_LIMIT}글자를 초과하여 입력할 수 없습니다.")
```

그 다음으로 이어지는 유효성 검증 로직도 마찬가지로 동일하게 처리해주세요! 🙏

Reply:
무심코 View에 있는 에러메시지를 RacingCar로 옮겼는데, 도메인이 View에 의존하게 되었네요.  
도메인에서 발생하는 에러 메시지를 콘솔에 출력하지 않도록 변경할게요!